src/components/GlobalStyles.js

```diff
@@ -19,6 +19,8 @@ export default function GlobalStyles({ children }) {
             background: #111;
             color: #fff;
             font-family: "Solway", serif;
+            --gradient: linear-gradient(101deg, #78e4ff, #ff48fa);
+            --gradientSecondary: linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
```

Comment:
If there is no good reason that the variables are in `html`, it's a good practice to add the variables to the `:root` pseudo class.
See https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#Basic_usage

Reply:
Moved the variables to root